placeholder-service/main/java/com/f4k0/placeholder/maps/MapsClient.java

```diff
@@ -0,0 +1,89 @@
+package com.f4k0.placeholder.maps;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.reactive.function.client.WebClient;
+import place.Place;
+import place.PlaceList;
+
+import java.util.Optional;
+
+public class MapsClient {
+
+    private static final int RADIUS = 500;
+    private static final Logger LOGGER = LoggerFactory.getLogger(MapsClient.class);
+    private static final String GMAPS_BASE_URL = "https://maps.googleapis.com/maps/api/place/";
+    private static final String GMAPS_KEY = System.getenv("GMAPS_API_KEY");
```

Comment:
This won't be accessible from the PR build. We need to find another solution to store the API_KEY.